### ISSUE TITLE
Header grid line of >1px produces vertical scrollbar

### ISSUE THREAD

**Issue by nicolasr75** [vaadin/vaadin-grid#874]
I am using the 2.1-polymer.2 branch and have trouble with the vertical scrollbar.

My header cell style is:

	--vaadin-grid-header-cell: {
		border-bottom: 1px solid blue;	
	};

This works as expected. However, I actually want to give the header cells a border-bottom of 2px to emphasize it but setting it to 2px produces a vertical scrollbar in the grid. Is this to be expected? Any idea what I can do about it?

**Comment by manolo** [comment]
I'm unable to reproduce the issue. I tried this [jsfiddle](https://jsfiddle.net/mcdodot/mwrroLm6/) 

Can you try your code with latest release? or provide an example with the problem ?

**Comment by nicolasr75** [comment]
Hello and thanks for looking into this. The latest release does not fix it for me. I wanted to change your fiddle to show the problem. The pre-condition is that the number of items is less than fits into the available space such that no scrollbars are needed. Unfortunately I don't know how to create an array manually in a dom-bind. Everything I tried did not work. Could you change the fiddle to not use the iron-ajax but simply create an array with just one person? Then I could try again...

**Comment by manolo** [comment]
Changed to show 3 users, you just need to play with the url parameters `url="https://demo.vaadin.com/demo-data/1.0/people?count=3"` is that enough?

**Comment by nicolasr75** [comment]
Sorry, I could have seen the people?count myself...

Now I have created my own [jsFiddle](https://jsfiddle.net/NicolasR75/aLsrvyef/4/)
I can not yet reproduce the issue though but two differences I see are that you are using Polymer 1.9.0 while in my project I was using one of the v2.0.0-rc.X versions and now am using 2.0.0. The other thing is that I was trying to set the grid to height 100% of the container.

In my fiddle I tried to use Polymer 2.0.0 but that does not seem to work at all. Any idea what I am doing wrong? The release page on Github says that 2.0.0 is the latest release, I also tried v2.0.0-rc.X etc but nothing works except v1.9.0 and v1.9.1.
